packages/liferay-npm-scripts/src/scripts/test.js

```diff
@@ -28,8 +29,29 @@ module.exports = function(arrArgs = []) {
 			buildSoy();
 		}
 
+		const env = {
+			...process.env,
+			NODE_ENV: 'test'
+		};
+
+		if (process.env.NODE_ENV !== 'test') {
+			log('Using NODE_ENV: "test"');
```

Comment:
Should we log this anyways? I think that's what we do in the `build` script?

Reply:
In the build script we log either:

    Using pre-existing NODE_ENV: "foo"

or:

    Using NODE_ENV: "production"

Reason I didn't log here if already "test" is just to keep the "noise" down, but I guess that's pointless and it won't hurt to log all the time. (If we do what I suggested earlier and stop passing "test"/"production" explicitly unless the user sets a concrete `nodejs.node.env`, then we'll end up logging all the time anyway.)